.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
just thinking should we include gomod as well?

Reply:
What do you think if PR will have ```chore``` prefix too